### DISCUSSION TITLE
elastic: configurable default-metric in explore-mode

### DISCUSSION THREAD

**Discussion by gabor** [grafana/grafana#61856] [Data Sources]
originally asked in https://github.com/grafana/grafana/discussions/37465 , a way to configure the elasticsearch datasource to start with `metric=logs`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82812. Feel free to continue the discussion around this there. Thank you!
